### PR TITLE
Updated BERTweet model card.

### PR DESCRIPTION
**Pull request by RogerSinghChugh**
# What does this PR do?
As mentioned in the issue https://github.com/huggingface/transformers/issues/36979 this PR updates the documentation of the BigBird model, which will now be aligned with the standardized format for all the docs.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@stevhliu

### CODE REVIEW

File: docs/source/en/model_doc/bertweet.md [hunk 1]
```diff
@@ -16,60 +16,84 @@ rendered properly in your Markdown viewer.
...
 </div>
 
-## Overview
+## BERTweet
 
-The BERTweet model was proposed in [BERTweet: A pre-trained language model for English Tweets](https://www.aclweb.org/anthology/2020.emnlp-demos.2.pdf) by Dat Quoc Nguyen, Thanh Vu, Anh Tuan Nguyen.
+[BERTweet](https://huggingface.co/papers/2005.10200) is basically BERT, but for tweets. It shares the same architecture as BERT-base, but it’s been pre-trained specifically on English Twitter data using the RoBERTa-style training setup. That means it’s way more familiar with tweet lingo — hashtags, abbreviations, usernames, and all the quirks of social media.
 
-The abstract from the paper is the following:
+Because of this Twitter-native training, BERTweet performs really well on tweet-related NLP tasks like part-of-speech tagging, named entity recognition, and text classification — even outperforming big models like RoBERTa-base and XLM-R-base in those areas.
```
Comment (stevhliu) [inline docs/source/en/model_doc/bertweet.md:31]:
```suggestion
```

File: docs/source/en/model_doc/bertweet.md [hunk 2]
```diff
@@ -16,60 +16,84 @@ rendered properly in your Markdown viewer.
...
-*We present BERTweet, the first public large-scale pre-trained language model for English Tweets. Our BERTweet, having
-the same architecture as BERT-base (Devlin et al., 2019), is trained using the RoBERTa pre-training procedure (Liu et
-al., 2019). Experiments show that BERTweet outperforms strong baselines RoBERTa-base and XLM-R-base (Conneau et al.,
-2020), producing better performance results than the previous state-of-the-art models on three Tweet NLP tasks:
-Part-of-speech tagging, Named-entity recognition and text classification.*
+You can find all the original BERTweet checkpoints under the [VinAI Research](https://huggingface.co/vinai?search_models=BERTweet) organization.
 
-This model was contributed by [dqnguyen](https://huggingface.co/dqnguyen). The original code can be found [here](https://github.com/VinAIResearch/BERTweet).
+> [!TIP]
+> This implementation is the same as BERT, except for tokenization method. Refer to [BERT](https://huggingface.co/docs/transformers/en/model_doc/bert) documentation for API reference information.
```
Comment (stevhliu) [inline docs/source/en/model_doc/bertweet.md:36]:
```suggestion
> Refer to the [BERT](./bert) docs for more examples of how to apply BERTweet to different language tasks.
```

File: docs/source/en/model_doc/bertweet.md [hunk 3]
```diff
@@ -16,60 +16,84 @@ rendered properly in your Markdown viewer.
...
+```
+</hfoption>
+<hfoption id="AutoModel">
+
+```py
+import torch
+from transformers import AutoModelForMaskedLM, AutoTokenizer
+
+tokenizer = AutoTokenizer.from_pretrained(
+   model_id,
```
Comment (stevhliu) [inline docs/source/en/model_doc/bertweet.md:63]:
```suggestion
   "vinai/bertweet-base",
```

File: docs/source/en/model_doc/bertweet.md [hunk 4]
```diff
@@ -16,60 +16,84 @@ rendered properly in your Markdown viewer.
...
->>> with torch.no_grad():
-...     features = bertweet(input_ids)  # Models outputs are now tuples
+</hfoption>
+<hfoption id="transformers CLI">
 
->>> # With TensorFlow 2.0+:
->>> # from transformers import TFAutoModel
->>> # bertweet = TFAutoModel.from_pretrained("vinai/bertweet-base")
+```bash
+echo -e "Plants create <mask> through a process known as photosynthesis." | transformers-cli run --task fill-mask --model vinai/bertweet-base --device 0
```
Comment (stevhliu) [inline docs/source/en/model_doc/bertweet.md:87]:
```suggestion
echo -e "Plants create <mask> through a process known as photosynthesis." | transformers run --task fill-mask --model vinai/bertweet-base --device 0
```

File: docs/source/en/model_doc/bertweet.md [hunk 5]
```diff
@@ -16,60 +16,84 @@ rendered properly in your Markdown viewer.
...
-<Tip> 
-
-This implementation is the same as BERT, except for tokenization method. Refer to [BERT documentation](bert) for 
-API reference information.  
+</hfoption>
+</hfoptions>
 
-</Tip>
+## Notes
+- Use the AutoTokenizer or BertweetTokenizer — it’s preloaded with a custom vocabulary adapted to tweet-specific tokens like hashtags (#), mentions (@), emojis, and common abbreviations.
```
Comment (stevhliu) [inline docs/source/en/model_doc/bertweet.md:94]:
```suggestion
- Use the [`AutoTokenizer`] or [`BertweetTokenizer`] because it’s preloaded with a custom vocabulary adapted to tweet-specific tokens like hashtags (#), mentions (@), emojis, and common abbreviations. Make sure to also install the [emoji](https://pypi.org/project/emoji/) library.
```

File: docs/source/en/model_doc/bertweet.md [hunk 6]
```diff
@@ -16,60 +16,84 @@ rendered properly in your Markdown viewer.
...
+- Inputs should be padded on the right (e.g., padding="max_length") to align with the positional encoding behavior.
```
Comment (stevhliu) [inline docs/source/en/model_doc/bertweet.md:95]:
```suggestion
- Inputs should be padded on the right (`padding="max_length"`) because BERT uses absolute position embeddings.
```

File: docs/source/en/model_doc/bertweet.md [hunk 7]
```diff
@@ -16,60 +16,84 @@ rendered properly in your Markdown viewer.
...
+- Since it was pre-trained on tweets, it performs best on short, informal, and social media-style text.
```
Comment (stevhliu) [inline docs/source/en/model_doc/bertweet.md:96]:
```suggestion
```

**Review by stevhliu — COMMENTED** [review]
Thanks! 🤗 

Let's also update the entry in the toctree from `Bertweet` to `BERTweet`.

File: docs/source/en/model_doc/bertweet.md (hunk 4 above)
Comment (RogerSinghChugh) [reply docs/source/en/model_doc/bertweet.md:86]: Hi @stevhliu, in a previous PR (for BART) you mentioned that I should leave the transformers-cli cmd as is, should I do that here as well? I might have mis understood what you meant.

Comment (stevhliu) [reply docs/source/en/model_doc/bertweet.md:86]: Yeah sorry, this should still be `transformers-cli run ...`

**Review by stevhliu — APPROVED** [review]
Thanks, once the `transformers-cli run...` command is fixed we can merge! Remember to fix the entry in the toctree from `Bertweet` to `BERTweet`

**Comment by RogerSinghChugh** [comment]
> `Bertweet`

Hi @stevhliu , should I update toctree of all languages?

**Comment by stevhliu** [comment]
Thanks, can you also push an empty commit here to trigger `run_tests` please?
